dashboard/config/initializers/markdown_handler.rb

```diff
@@ -6,8 +7,29 @@ module MarkdownHandler
     space_after_headers: true
   }
 
+  # Rewrite youtube iframe elements to use the fallback-player iframe instead
+  class YoutubeRewriter < Redcarpet::Render::HTML
+    def block_html(html)
+      doc = ::Nokogiri::HTML(html)
+      nodes = doc.xpath(%w(youtube youtubeeducation).map{|x| "//iframe[@src[contains(.,'#{x}.com/embed')]]"}.join(' | '))
+      nodes.each do |node|
+        if node['src']
+          regex = /(http:|https:)?\/\/(www\.)?(youtube|youtubeeducation)\.com\/embed\/([^"&?\/ ]{11})/
```

Comment:
Please explain the magic number 11 (e.g. by making it a constant or computing it somehow.)